packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -541,6 +555,27 @@ test.serial('C25 - LPs can deposit USDC', async t => {
   t.deepEqual(poolBalance, make(usdc.brand, 250_000_000n + balance0.value));
 });
 
+test.serial('Contract skips advance when risks identified', async t => {
+  const {
+    common: {
+      commonPrivateArgs: { feeConfig },
+      utils: { transmitTransferAck },
+    },
+    evm: { cctp, txPub },
+    startKit: { metricsSub },
+    bridges: { snapshot, since },
+    mint,
+  } = t.context;
+  const custEmpty = makeCustomer('Skippy', cctp, txPub.publisher, feeConfig);
+  const bridgePos = snapshot();
+  const sent = await custEmpty.sendFast(t, 1_000_000n, 'osmo123', true);
+  const bridgeTraffic = since(bridgePos);
+  await mint(sent);
+  custEmpty.checkSent(t, bridgeTraffic, 'forward');
+  t.log('No advancement, just settlement');
+  await transmitTransferAck(); // ack IBC transfer for forward
```

Comment:
somewhere with more integration than the exo tests we should again cover vstorage output. But I think a bootstrap test is the right place. That's pretty much an "acceptance" test wrt product spec.

Reply:
Sounds good, added a bootstrap test.